spring-cloud-gcp-data-firestore/src/main/java/com/google/cloud/spring/data/firestore/FirestoreTemplate.java

```diff
@@ -427,7 +434,7 @@ private void doIfTransaction(Context ctx, Consumer<ReactiveFirestoreResourceHold
 		return buildResourceName(persistentEntity, idVal.toString());
 	}
 
-	private String buildResourceName(FirestorePersistentEntity<?> persistentEntity, String s) {
+	public String buildResourceName(FirestorePersistentEntity<?> persistentEntity, String s) {
```

Comment:
If we make this public, it should have a javadoc.

Reply:
Good point, done.